artifactory/services/props.go

```diff
@@ -44,21 +44,11 @@ func (ps *PropsService) GetThreads() int {
 }
 
 func (ps *PropsService) SetProps(propsParams PropsParams) (int, error) {
-	log.Info("Setting properties...")
```

Comment:
I think all the logs in this file are useful, why don't you like them?

Reply:
setProps calls put/delete props and they already have a log of their own:

- log.Info(logMsgPrefix+"Deleting properties on:", relativePath)
- log.Info(logMsgPrefix+"Setting properties on:", relativePath)

@RobiNino